contents/handbook/people/team-structure/team-structure.md

```diff
@@ -44,34 +44,34 @@ We've organised the team into small teams that are multi-disciplinary. [You can
 
 ### Engineering
 
-#### [Core Analytics](core-analytics)
-- [Eric Duong](/handbook/company/team#eric-duong-software-engineer) (Team lead, Full Stack Engineer)
-- [Neil Kakkar](/handbook/company/team#neil-kakkar-software-engineer) (Full Stack Engineer)
-- [Li Yi Yu](/handbook/company/team#li-yi-yu-full-stack-engineer) (Full Stack Engineer)
+#### Team East
```

Comment:
I get this is for people in same timezone, but this name makes it impossible to know what they'd be working on from the name alone, making it harder for people one step removed to give feedback/connect to users etc where relevant.

Reply:
There's quite a lot of context missing here, however other than just the East/West split, we also assigned [clear owners for each feature area](https://github.com/PostHog/posthog.com/pull/3020) - this should make it easier to connect to the right person. If that person is unavailable for whatever reason, the request should be handled by someone else inside the team.

Reply:
I've added more context to explain this. Basically we decided to spread teams in a way that each team has full expertise to work on any areas of the product, so the split is no longer based on scope. That being said, the feature ownership (https://posthog.com/handbook/engineering/feature-ownership) should provide context on who's working on what.

Let me know if this clarifies it for someone without context.

Reply:
That makes it clearer thanks!